templates/case/event/copy.html.twig

```diff
@@ -0,0 +1,33 @@
+{% extends 'layout-with-case-submenu.html.twig' %}
+
+{% trans_default_domain 'case_event' %}
+
+{% block title %}{% trans %}Copy case event to other cases{% endtrans %}{% endblock %}
+
+{% block header %}
+    <div class="row">
+        <div class="col">
+            <h1>{% trans %}Copy case event{% endtrans %}</h1>
+        </div>
+        <div class="col-auto">
+            <a class="btn btn-primary" href="{{ path('case_event_show', {id: case.id, caseEvent: case_event.id}) }}">{% trans %}Cancel{% endtrans %}</a>
+        </div>
+    </div>
+{% endblock %}
+
+{% block content %}
+    <main role="main" class="col-12 mt-3">
+        {{ form_start(form) }}
+        <div>
+            <label >{% trans %}Cases{% endtrans %}</label>
+            {{ form_widget(form.cases) }}
+        </div>
+        <button class="btn btn-success float-right mt-3">{{ button_label|default('Copy'|trans([], 'case_event', 'da')) }}</button>
+        {{ form_end(form) }}
+    </main>
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+    {{ encore_entry_script_tags('case_event_copy_select2') }}
```

Comment:
`_select2` should not be part of this name.

Reply:
Removed.